### ISSUE TITLE
Incorrect Animations with Full Hotbar with Villager Trades

### ISSUE THREAD

**Issue by Adamiok** [LOOHP/InteractionVisualizer#104]
## Description
In minecraft, you can swap an itemstack between a slot in your hotbar and the slot where your mouse is. In the case that the inventory you want to swap to is read and remove only (like the villager trade result), and the hotbar slot is empty the item will just move. However in the case that the hotbar slot is full, no action is done.

## Bugged Actions
When the hotbar slot is full and the hotbar key is pressed, no trade occurs. However the trade animation is still played.

## Expected Solution
When the trade does not happen (because the hotbar slot is full) no animation should play.

## Steps to reproduce
1. Get a 1.20 paper server, install InteractionVisualizer and ProtocolLib.
2. Join the server with a 1.20 client
3. Get a villager spawn egg and place it
4. Give the villager a workstation
5. Wait for villager to accept workstation
6. Place a item, which is not traded by the villager, into your first hotbar slot (eg. dirt block)
7. Get a stack of emeralds
8. Open the villager trade GUI
9. On the trade selector, purchase an item with the emeralds, BUT don't take it from the trade output slot
10. Hover your mouse over the trade output slot
11. Press the key '1'
12. Observe how a trade did not happen but animation was played.

Just a small issue, otherwise great plugin!
**TL;DR: Minecraft villagers display trade animations even when a trade fails due to a full hotbar.**

**Comment by LOOHP** [comment]
Thank you for the detailed description and clear steps to reproduce!

**Comment by Adamiok** [comment]
Thanks for the quick fix!
